src/rules/tripleEqualsRule.ts

```diff
@@ -55,7 +60,10 @@ class ComparisonWalker extends Lint.RuleWalker {
     private isExpressionAllowed(node: ts.BinaryExpression) {
         const nullKeyword = ts.SyntaxKind.NullKeyword;
 
-        return this.hasOption(OPTION_ALLOW_NULL_CHECK)
-            && (node.left.kind ===  nullKeyword || node.right.kind === nullKeyword);
+        return (
+            this.hasOption(OPTION_ALLOW_NULL_CHECK) && (node.left.kind ===  nullKeyword || node.right.kind === nullKeyword)
```

Comment:
Seems like there's an extra space here: `===  nullKeyword`. Looks like it was here before too, but if you're tweaking this file, feel free to remove it :)